src/containers/shared/components/Transaction/NFTokenAcceptOffer/parser.ts

```diff
@@ -1,13 +1,22 @@
-const formatAmount = require('./formatAmount')
+import { NFTokenAcceptOffer, NFTokenAcceptOfferInstructions } from './types'
+import { TransactionParser } from '../types'
+import formatAmount from '../../../../../rippled/lib/txSummary/formatAmount'
 
-module.exports = (tx, meta) => {
+export const parser: TransactionParser<
+  NFTokenAcceptOffer,
+  NFTokenAcceptOfferInstructions
+> = (tx, meta) => {
   const acceptedOfferNode = meta.AffectedNodes.find(
-    (node) => node.DeletedNode?.LedgerEntryType === 'NFTokenOffer',
+    (node: any) => node.DeletedNode?.LedgerEntryType === 'NFTokenOffer',
   )?.DeletedNode?.FinalFields
 
-  const acceptedOfferIDs = [tx.NFTokenBuyOffer, tx.NFTokenSellOffer].filter(
```

Comment:
Does type checking not like this.

Reply:
Yes, cause [string|undefined, string|undefined].filter is still potentially undefined